Site/src/Site.Web/comp-site/src/app/components/team-hub/github-account/github-account.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit } from '@angular/core';
+import { UserSessionService } from 'src/app/services/user-session.service';
+
+@Component({
+  selector: 'app-github-account',
+  templateUrl: './github-account.component.html',
+  styleUrls: ['./github-account.component.scss']
+})
+export class GithubAccountComponent implements OnInit {
+  
+
+  repo: string = '';
+
+  get accountLinked() {
+    return false;
```

Comment:
Should this be a variable?

Reply:
Maybe will be yeah, not got the backend for this yet.